### PR TITLE
Flutter format added to CI

### PR DESCRIPTION
**Pull request by theswerd**
This PR gets rid of 2 unused files, and an old flutter test that was testing files that are not used in the current app. It provides a new flutter test as an example, and adds flutter format to the CI

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).
- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR? 
Closing #334
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [ ] Included any relevant screenshots of UI updates. DOESN'T APPLY
- [X] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [X] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [X] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by SamMousa** [comment]
Lgtm except for the naming of the ci jobs

File: .github/workflows/client-format.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - 'api/**'
+      - 'client/flutter/**'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - '.github/workflows/**'
+      - 'api/**'
+      - 'client/flutter/**'
+jobs:
+  test:
```
Comment (SamMousa) [inline .github/workflows/client-format.yaml:15]: This should be renamed

File: .github/workflows/client-format.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    name: Test client on ${{ matrix.os }}
```
Comment (SamMousa) [inline .github/workflows/client-format.yaml:16]: Same

**Comment by advayDev1** [comment]
Hunter's thinking through a rationalization of our automation.  Want to make sure he gets a chance to TAL.

My 2cents, `flutter format` should be part of our Client Tests.  An extra action is costly in terms build minutes and dev-waiting-around-time.

**Comment by SamMousa** [comment]
Actually actions run in parallel, our minutes are free. Due to the efficient caching this can work as well.

Regardless of our process it is important that we merge this fast since it will need to be updated with any other commits from master that need to be formatted too.

**Comment by SamMousa** [comment]
I'd propose merging and then hunter can always change the CI side of things.

**Comment by theswerd** [comment]
@SamMousa I can revert a to a push ago if you want them separate, and keep this one if you want them together.

Take your pick

**Comment by advayDev1** [comment]
@SamMousa @theswerd that is fine by me.  Btw the current error:
`(CDN: trunk URL couldn't be downloaded: https://cdn.cocoapods.org/CocoaPods-version.yml Response: Timeout was reached)`

is one reason to have fewer actions.  Less times we redownload these things and can have errors.


I'm fine with this until Hunter has a chance to look deeper.

**Pull request merged**